packages/console/src/components/Entities/EntityDetails.tsx

```diff
@@ -57,58 +63,70 @@ interface EntityDetailsProps {
  */
 export const EntityDetails: React.FC<EntityDetailsProps> = ({ id }) => {
   const sections = entitySections[id.resourceType];
-  const project = useProject(id.project);
+  const [project] = useProject(id.project);
   const styles = useStyles();
   const { chartIds, onToggle, clearCharts } = useChartState();
 
   return (
-    <WaitForData {...project}>
-      <EntityDetailsHeader
-        project={project.value}
-        id={id}
-        launchable={!!sections.launch}
-      />
+    <Grid container direction="column" className={styles.entityDetailsWrapper}>
+      {project?.id ? (
+        <>
+          <EntityDetailsHeader
+            project={project}
+            id={id}
+            launchable={!!sections.launch}
+          />
 
-      <div className={styles.metadataContainer}>
-        {sections.description ? (
-          <div className={styles.descriptionContainer}>
-            <EntityDescription id={id} />
-          </div>
-        ) : null}
-        {!sections.inputs && sections.schedules ? (
-          <div className={styles.schedulesContainer}>
-            <EntitySchedules id={id} />
+          <div className={styles.metadataContainer}>
+            {sections.description ? (
```

Comment:
can be made more concise

sections.description && (
<div className={styles.descriptionContainer}>
                <EntityDescription id={id} />
              </div>
)